src/run.jl

```diff
@@ -64,7 +64,7 @@ function test(dir, tmp, idx; dead = false)
   return !pass
 end
 
-function go(dir, ps = defaults; procs = 1, dead = false)
+function go(dir, ps = defaults; procs = 1, dead = false, tests = Inf)
```

Comment:
I would just use `typemax(Int)` here so that this function doesn't get generated twice when it gets called with a different type for `tests`.